docs/classad-attributes/accounting-classad-attributes.rst

```diff
@@ -11,100 +11,60 @@ to see the full set of raw accounting ads, run the command:
     The total number of seconds this submitter has used since they first
     arrived in the pool.  Note this is not weighted by cpu cores -- an
     eight core job running for one hour has a usage of 3600, compare with 
     WeightedAccumulatedUsage
 
-:index:`BeginUsageTime`
-:index:`BeginUsageTime<single: Accounting ClassAd attribute; BeginUsageTime>`
-
-``BeginUsageTime``:
+:classad-attribute:`BeginUsageTime`
     The Unix epoch time in seconds when this user claimed resources in the system.
     This is persistent and survives reboots and HTCondor upgrades.
 
-:index:`ConfigQuota`
-:index:`ConfigQuota<single: Accounting ClassAd attribute; ConfigQuota>`
- 
-``ConfigQuota``:
+:classad-attribute:`ConfigQuota`
     If this record is for an accounting group with quota, the amount of quota
     statically configured.
 
-:index:`IsAccountingGroup`
-:index:`IsAccountingGroup<single: Accounting ClassAd attribute; IsAccountingGroup>`
-
-``IsAccountingGroup``:
-    A boolean which is true if this record represents an accounting group
-
-:index:`LastUsageTime`
-:index:`LastUsageTime<single: Accounting ClassAd attribute; LastUsageTime>`
-
-``LastUsageTime``:
+:classad-attribute:`IsAccountingGroup`
     The unix epoch time, in seconds, when this submitter last had
     claimed resources.
 
-:index:`Priority`
-:index:`Priority<single: Accounting ClassAd attribute; Priority>`
+:classad-attribute:`LastUsageTime`  
+    The unix epoch time, in seconds, when this submitter last had
+    claimed resources.
 
-``Priority``:
+:classad-attribute:`Priority`
     The current effective priority of this user.
 
-:index:`PriorityFactor`
-:index:`PriorityFactor<single: Accounting ClassAd attribute; PriorityFactor>`
-
-``PriorityFactor``:
+:classad-attribute:`PriorityFactor`
     The priority factor of this user.
 
-:index:`ResourcesUsed`
-:index:`ResourcesUsed<single: Accounting ClassAd attribute; ResourcesUsed>`
-
-``ResourcesUsed``:
+:classad-attribute:`ResourcesUsed`
     The current number of slots claimed.
 
-:index:`SubmitterShare`
-:index:`SubmitterShare<single: Accounting ClassAd attribute; SubmitterShare>`
-
-``SubmitterShare``:
+:classad-attribute:`SubmitterShare`
     When the negotiator computes the fair share of the pool that
     each user should get, assuming they have infinite jobs and every job
     matches every slot, the SubmitterShare is the fraction of the pool
     this user should get.  A floating point number from 0 to 1.0.
 
-:index:`SubmitterLimit`
-:index:`SubmitterLimit<single: Accounting ClassAd attribute; SubmitterLimit>`
-
-``SubmitterLimit``:
+:classad-attribute:`SubmitterLimit`
     When the negotiator computes the fair share of the pool that
     each user should get, assuming they have infinite jobs and every job
     matches every slot, the SubmitterLimit is the absolute number of cores
     this user should get.
 
-:index:`Name<single: Accounting ClassAd attribute; Name>`
-
-``Name``:
+:classad-attribute:`Name`
```

Comment:
This is a small-enough page that I wonder if it shouldn't be alphabetized, which would only require moving the `Name` entry up before `Priority`.